libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Util/SFManagedPreferences.m

```diff
@@ -63,6 +63,10 @@ + (instancetype)sharedPreferences {
     dispatch_once(&pred, ^{
         preferences = [[self alloc] init];
     });
+    
+    dispatch_once(&pred, ^{
```

Comment:
This is not doing what you think.  Given that `pred` is "exhaused" in the prior `dispatch_once` call, the second call is not being made at all.